site/frontend/src/pages/status/collector.vue

```diff
@@ -101,6 +117,35 @@ function formatProfile(job: BenchmarkJob): string {
     return "";
   }
 }
+function timeSince(timestamp: string): string {
```

Comment:
We have a `site/frontend/src/utils/formatting.ts` file, I think this should live there

Reply:
I usually only put functions into a shared module if more than a single file needs them. Those two functions (especially `formatTime`) are quite specific for this file, and I don't think that they belong to `formatting.ts` without being generalized somehow.